README.md

```diff
@@ -1,2 +1,44 @@
-# flask
-Starting a basic Python Flask server
+# flask - project
```

Comment:
Move whole project to a folder `DEVOPS_PROJECTS` and recommending `Flask_Web_Framework` as project name

Reply:
Include codebase you have used along with the readme file